src/commands/utils/extensions.ts

```diff
@@ -0,0 +1,17 @@
+import * as vscode from 'vscode';
```

Comment:
as discussed this file name can be utils/extensionHelpers.ts.ts or extensionHandlers.ts.

Reply:
This is removed, made changes in previous PR